docs/modules/ROOT/pages/howtos/quickstart_k3s_docker.adoc

```diff
@@ -29,12 +29,35 @@ $ terraform apply
 
 You should see the services URL as Terraform outputs.
 
-== Get kubeconfig
+== Get kubeconfig and admin password
+
+Retrieve the Kubeconfig file:
 
 ```shell
 $ terraform output -json kubeconfig | jq -r .
 ```
 
+
+Retrieve the Keycloak admin password:
```

Comment:
well, technically it's not the Keycloak admin password but the password of the admin user to connect to applications connected to Keycloak.

Reply:
Yes. Also, technically, we can retrieve it from the `keycloakuser` resource, no need for an extra output variable.

Reply:
Something like `kubectl -n keycloak get keycloakusers.keycloak.org admin --template='{{(index .spec.user.credentials 0).value}}'`